polling_stations/apps/file_uploads/management/commands/check_pending.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+from django.conf import settings
+from django.utils.timezone import now
+from file_uploads.models import Upload
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        from_time = now() - timedelta(minutes=5)
```

Comment:
Is there a reason for the 5 minute cut off. I guess I expect it's enough, but if we started doing more stuff it might not be. given we'll probably run this management command hourly I'd be more inclined to have a 15 minute cut off, to reduce the chance of false positives

Reply:
@pmk01 Any reason for 5 mins?

Reply:
I agree. I picked 5 minutes out of the air, as the current Lambda should have finished within that time.

The [function timeout is 15 minutes](https://github.com/DemocracyClub/wdiv-s3-trigger/blob/master/serverless.yml#L10) though, so we should set this check to `$timeout + [some padding]`

Reply:
Yep, no reason for 5 minutes, can be any length of time.